docker-compose.yaml

```diff
@@ -29,6 +43,26 @@ services:
         condition: service_healthy
       orchestrator:
         condition: service_healthy
+      # Make sure this doesn't start until the other contracts have been deployed, since we use the same mnemonic.
+      deploy-sequencer-contracts:
+        condition: service_completed_successfully
+
+  fund-builder:
+    image: ghcr.io/espressosystems/espresso-sequencer/bridge:main
```

Comment:
Changes for the new docker image are missing in this PR.

Reply:
oops, thanks

Reply:
Added in [97fdcf1](https://github.com/EspressoSystems/espresso-sequencer/pull/1390/commits/97fdcf175d02e15a6a42f40e32d1cc4f69c9adae)